framework/src/meshgenerators/PatternedMeshGenerator.C

```diff
@@ -76,8 +76,36 @@ PatternedMeshGenerator::generate()
   // Reserve spaces for all the meshes
   _meshes.reserve(_input_names.size());
 
+  // Getting the boundaries provided by the user
```

Comment:
Remember that even with that check in place, I still found a way to break the mesh generator without triggering the check. Because we don't have a way to catch these cases (without normals checking), I don't see a reason to leave the check in place since there are valid cases 1 > num_boundaries > 4 that produce correct meshes. Still want me to add the check back in?

Reply:
Also, I did update the manual page for PatternedMeshGenerator to inform the user specifically what to look for when the mess up their inputs.

Reply:
As you want. I think it s a fine limitation

Reply:
Will leave the check out.